src/clr-angular/forms/styles/_mixins.forms.scss

```diff
@@ -90,6 +90,7 @@
     //and so that the user's default input size doesn't affect it
     height: $size;
     width: $size;
+    display: none;
```

Comment:
@adityarb88: Wasn't `display: none` a blocker for accessibility? I don't remember it it was screenreaders that couldn't work with it.

Reply:
Yes `display: none` is a blocked on accessibility.  I believe we used `opacity: 0` before. @gnomeontherun why did we change this to `display: none`? Are you still able to checkbox of the input using the keyboard?

Reply:
This is a mistake, Firefox was still calculating a size for it so the tests failed. Will fix.